.github/workflows/golangci-lint.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           # When updating the version, update the update-linters target in Makefile as well.
           version: v1.57.2
           args: --verbose --timeout 10m --fix=false --new-from-rev=HEAD~ --config=.golangci.yml
+          skip-pkg-cache: true
```

Comment:
As discussed internally, the lint step was causing problems regularly - this seems to fix it.